### PR TITLE
gitlab-ci-multi-runner 1.5.3

### PR DESCRIPTION
**Pull request by ilovezfs**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/gitlab-ci-multi-runner.rb [hunk 1]
```diff
@@ -4,8 +4,8 @@ class GitlabCiMultiRunner < Formula
   desc "The official GitLab CI runner written in Go"
   homepage "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner"
   url "https://gitlab.com/gitlab-org/gitlab-ci-multi-runner.git",
-    :tag => "v1.5.2",
-    :revision => "76fdacdf55c5a83249a4bdc5bd8c650e7a5ff796"
+    :tag => "v1.5.3",
+    :revision => "fb49c4781e5040539d3faec87a2a63675d2c9f00"
```
Comment (zmwangx) [inline Formula/gitlab-ci-multi-runner.rb]: One day I'll align these to the opening double quote. I swear. 👿

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: now one fewer on your plate

Comment (zmwangx) [inline Formula/gitlab-ci-multi-runner.rb]: What, you just broke consistency with everything else... 😮

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: no there are several like this and several the other way

Comment (zmwangx) [inline Formula/gitlab-ci-multi-runner.rb]: Okay, guess there's a selection bias somewhere.

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: For urls everything that wraps to the next line should be transitioned to align under the `"` just as you did with the go_resource blocks. The only thing that I think remains controversial is which elements should not wrap at all. I'd like to see us do one element per line regardless of which subset of elements it is. So `:shallow` gets its own line, `:branch` gets its own line, `:tag` gets its own line, `:revision` gets its own line, `:using` gets it own line. But I think @DomT4 wants to make `:branch` an except that never wraps, and I think possibly `:shallow` too, but I'm not sure. And then other exceptions creep in like if the `:revision` doesn't cause it to exceed the eightieth column then you may (must? should?) keep it on the same line, etc., as well as possibly other exceptions that have yet to be revealed.

Comment (zmwangx) [inline Formula/gitlab-ci-multi-runner.rb]: Placing anything at the end of a long line (that is long solely due to a long verbatim string) is probably a bad idea. Who would read that.

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: exactly.

Comment (DomT4) [inline Formula/gitlab-ci-multi-runner.rb]: > But I think @DomT4 wants to make :branch an exception that never wraps, and I think possibly :shallow too, but I'm not sure.

It's not a hard rule. I'm fine with every `:` having its own line, I was just wary of bulking out formulae but eh.

Comment (ilovezfs) [inline Formula/gitlab-ci-multi-runner.rb]: So no :cold_sweat: then?

Comment (DomT4) [inline Formula/gitlab-ci-multi-runner.rb]: Na, if you want each `:` URL blob on its own line, go for it.

**Pull request closed without merging**